public/app/plugins/datasource/loki/querybuilder/LokiQueryModeller.ts

```diff
@@ -29,25 +29,136 @@ export class LokiQueryModeller extends LokiAndPromQueryModellerBase {
   getQueryPatterns(): LokiQueryPattern[] {
     return [
       {
-        name: 'Log query and label filter',
+        name: 'Log query with parsing',
+        // {} |= `` | logfmt | label="value"
         operations: [
-          { id: LokiOperationId.LineMatchesRegex, params: [''] },
+          { id: LokiOperationId.Logfmt, params: [] },
+          { id: LokiOperationId.LabelFilterNoErrors, params: [] },
+        ],
+      },
+      {
+        name: 'Log query with filtering and parsing',
+        // {} |= `` | logfmt | __error__=``
+        operations: [
+          { id: LokiOperationId.LineContains, params: [''] },
+          { id: LokiOperationId.Logfmt, params: [] },
+          { id: LokiOperationId.LabelFilterNoErrors, params: [] },
+        ],
+      },
+      {
+        name: 'Log query with parsing and label filter',
+        // {} |= `` | logfmt | __error__=''
+        operations: [
+          { id: LokiOperationId.LineContains, params: [''] },
           { id: LokiOperationId.Logfmt, params: [] },
           { id: LokiOperationId.LabelFilterNoErrors, params: [] },
           { id: LokiOperationId.LabelFilter, params: ['', '=', ''] },
         ],
       },
       {
-        name: 'Time series query on value inside log line',
+        name: 'Log query with parsing of nested json',
+        // {} |= `` | json | line_format `{{ .message}}` | json
         operations: [
-          { id: LokiOperationId.LineMatchesRegex, params: [''] },
+          { id: LokiOperationId.LineContains, params: [''] },
+          { id: LokiOperationId.Json, params: [] },
+          { id: LokiOperationId.LabelFilterNoErrors, params: [] },
+          { id: LokiOperationId.LineFormat, params: ['{{.message}}'] },
+          { id: LokiOperationId.Json, params: [] },
+          { id: LokiOperationId.LabelFilterNoErrors, params: [] },
+        ],
+      },
+      {
+        name: 'Log query with updated log line',
```

Comment:
```suggestion
        name: 'Log query with reformatted log line',
```
Suggestion since its more specific about what's being updated.